pkg/dev/podmandev/pod.go

```diff
@@ -44,20 +44,40 @@ func createPodFromComponent(
 			Name: storage.OdoSourceVolume,
 			VolumeSource: corev1.VolumeSource{
 				PersistentVolumeClaim: &corev1.PersistentVolumeClaimVolumeSource{
-					ClaimName: getVolumeName(componentName, appName, "source"),
+					ClaimName: getVolumeName(componentName, appName, "odo-projects"),
 				},
 			},
 		},
 		{
 			Name: storage.SharedDataVolumeName,
 			VolumeSource: corev1.VolumeSource{
 				PersistentVolumeClaim: &corev1.PersistentVolumeClaimVolumeSource{
-					ClaimName: getVolumeName(componentName, appName, "shared"),
+					ClaimName: getVolumeName(componentName, appName, "odo-shared"),
```

Comment:
```suggestion
					ClaimName: getVolumeName(componentName, appName, storage.SharedDataVolumeName),
```

Reply:
I am not sure when/where/how we use shared volume though. It is supposed to be the volume shared between InitContainer and the applicationContainer, but I don't know how init containers can be defined inside Devfile and used with `odo dev`.